lib/workout.js

```diff
@@ -17,11 +17,12 @@ module.exports = class Workout extends Content {
     this.description = '';
     this.section = null;
     this.type = '';
+    this.exercises = [];
 
     try {
```

Comment:
I think this is bad, having the parsing wrapped in a `try/catch` 
I'd say we remove this for all classes and let errors `throw` every time sometimes goes wrong. Otherwise, external applications (i.e. content consumer) can't possibly know when something goes wrong and handle that themselves.

In general, I think error handling should not happen within the parser.
I have these changes in my local copy of the repo, but never got to push them.

@lizTheDeveloper @nem035

Reply:
fixed this on this branch, feel free to file a PR with the rest of the changes :D